gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-5.6.3-bin.zip
```

Comment:
Any special reason we have `https:\` before? @bolobobo

Reply:
Are we using some special feature of gradle 5.6.3? @hiteshsharma

Reply:
@mchen07 seems `https\://` is broken url. Not sure how it worked. I simply copied the link from https://services.gradle.org/distributions/
No we are not using any 5.6.3 feature for now but someone created issue #231 for upgrading so I just upgraded it and it came out to be straightforward enough as we don't have any backwards incompatible usage

Reply:
@hiteshsharma  Thanks for pointing that out. I just followed the guideline provided by Gradle company. Please look at this doc. https://docs.gradle.org/current/userguide/gradle_wrapper.html I need to double check this with @mockitoguy .

Reply:
@hiteshsharma This file is auto generated by running ` ./gradlew wrapper` and that's probably for URI encoding. We should not edit that file by hand.